packages/blade/src/components/Table/docs/APIStories/VirtualizedTableAPI.stories.tsx

```diff
@@ -0,0 +1,156 @@
+import React from 'react';
```

Comment:
Let's combine stories as saurabh suggested?

Reply:
now we just have this story. i have removed others.